src/components/common/Header.tsx

```diff
@@ -10,14 +10,17 @@ import { categoryType } from '../../utils/constants/header';
 export default function Header() {
   const navigate = useNavigate();
   const { pathname } = useLocation();
+  const [clickedIdNum, setClickedIdNum] = useState<number>(2);
 
   const handlePageMove = (item: categoryType) => {
     if (item.id === 2) {
       navigate(`${item.name}`);
+      setClickedIdNum(item.id);
     }
 
     if (item.id === 3) {
       navigate(`${item.name}`);
+      setClickedIdNum(item.id);
     }
```

Comment:
내부 코드가 같으니 합치면 좋을 것 같아요~

```suggestion
    if(item.id === 2 || item.id === 3) {  
      navigate(`${item.name}`);
      setClickedIdNum(item.id);
    }
```